ufo2fontir/src/source.rs

```diff
@@ -1346,11 +1345,17 @@ mod tests {
         (source, input)
     }
 
-    fn build_static_metadata(name: &str) -> (impl Source, Context) {
+    fn flags_without_emit_ir() -> Flags {
+        Flags::default() - Flags::EMIT_IR
+    }
+
+    fn build_static_metadata(name: &str, flags: Flags) -> (impl Source, Context) {
```

Comment:
you could even make `flags: Option<Flags>` and when caller pass `None` it means "I don't care, just use the test defaults" (which in our case means `flags_without_emit_ir`), so that in future if we want to change the test default flags we can do in one place

Reply:
ok, you convinced me :)
can you maybe rename flags_without_emit_ir to something like default_flags_for_tests or similar?